pkg/networkpolicy/controller.go

```diff
@@ -67,6 +68,24 @@ type Config struct {
 	QueueID                    int
 	NodeName                   string
 	NetfilterBug1766Fix        bool
+	NFTableName                string // if other projects use this controllers they need to be able to use their own table name
+}
+
+func (c *Config) Defaults() error {
+	var err error
+	if c.QueueID == 0 {
+		c.QueueID = 100
+	}
+	if c.NodeName == "" {
+		c.NodeName, err = os.Hostname()
```

Comment:
sorry I am not familiar with this.. what is the NodeName here used for?

Reply:
The network policy implementation , for being more efficient, only apply policis to the Pods that are local (same node)

Reply:
a I see :) makes sense thanks!